### DISCUSSION TITLE
New tests style.

### DISCUSSION THREAD

**Discussion by SimpleFoxOfficial** [BEEmod/BEE2-items#4114] [Feature Suggestions]
F-Stop chambers style (from concept in final hours of portal 2)
![image](https://user-images.githubusercontent.com/82114463/164410225-e6f22330-e314-479f-989c-8d5c20ca5c97.png)
![image](https://user-images.githubusercontent.com/82114463/164410334-b8b6c695-7c36-45da-9b62-704d112e8467.png)

Thinking with time machine (mod)
![image](https://user-images.githubusercontent.com/82114463/164410467-a94d7b3b-7274-464b-96f2-97118a786b98.png)
![image](https://user-images.githubusercontent.com/82114463/164410523-417692d9-7691-4c6f-999a-e36a4843cede.png)

**Comment by TeamSpen210** [comment]
The beta F-Stop style would require huge numbers of custom resources to do. TWTM's style isn't that good with the portal surface confusion. The time would be better spent improving existing styles.

**Reply by slambo311** [reply]
The TWTM style might not be that good, but how about at least its custom test elements, such as upside-down droppers and monitors (with the timer controlling which video to play)?

**Reply by SimpleFoxOfficial** [reply]
And the obs rooms style

**Reply by SimpleFoxOfficial** [reply]
Also monitors can show the video, that showing on elevator
